### PR TITLE
Simplify ChunkTreeBuilder

### PR DESCRIPTION
**Pull request by rynowak**

### CODE REVIEW

File: src/Microsoft.AspNet.Razor/Chunks/ChunkTreeBuilder.cs [hunk 1]
```diff
@@ -29,11 +32,11 @@ public void AddChunk(Chunk chunk, SyntaxTreeNode association, bool topLevel = fa
             // If we're not in the middle of a parent chunk
             if (_parentStack.Count == 0 || topLevel == true)
```
Comment (NTaylorMullen) [inline src/Microsoft.AspNet.Razor/Chunks/ChunkTreeBuilder.cs]: Should fix this up. Looks like you undid the changes you previously made here.

**Comment by NTaylorMullen** [comment]
:shipit:

File: src/Microsoft.AspNet.Razor/Chunks/ChunkTreeBuilder.cs (hunk 1 above)
Comment (rynowak) [inline src/Microsoft.AspNet.Razor/Chunks/ChunkTreeBuilder.cs]: :+1:

Comment (rynowak) [inline src/Microsoft.AspNet.Razor/Chunks/ChunkTreeBuilder.cs]: :+1:

**Pull request closed without merging**